### PR TITLE
Implemented CreateMovie() and CreateProgram() methods

### PR DESCRIPTION
**Pull request by Jeckaa**
Waiting for @popovicjov #8  &  @magdalena997 #7

### CODE REVIEW

**Review by magdalena997 — DISMISSED** [review]
LGTM

File: MovieFestival/App.cs [hunk 1]
```diff
@@ -6,5 +6,39 @@ static void Main(string[] args)
...
+        return program;
+    }
+
+    public Movie CreateMovie()
+    {
+        string title = ReadUserInput("Insert movie title");
+        string length = ReadUserInput("Insert movie length");
+        string genreStr = ReadUserInput("Insert movie genre");
+        Genre genre = new Genre(genreStr);
+
```
Comment (stefand-code) [inline MovieFestival/App.cs:27]: Add integer check for length input.

**Review by stefand-code — CHANGES_REQUESTED** [review]
Add integer check for length input.

**Review by magdalena997 — APPROVED** [review]
LGFM

File: MovieFestival/App.cs (hunk 1 above)
Comment (Jeckaa) [reply MovieFestival/App.cs:27]: Not relevant

**Review by stefand-code — APPROVED** [review]

**Pull request merged**